src/cdk/table/table.ts

```diff
@@ -618,6 +620,8 @@ export class CdkTable<T> implements AfterContentChecked, CollectionViewer, OnDes
           (record) => record.item.data,
           (change: _ViewRepeaterItemChange<RenderRow<T>, RowContext<T>>) => {
             if (change.operation === _ViewRepeaterOperation.INSERTED && change.context) {
+              const rowView = <RowViewRef<T>>viewContainer.get(change.record.currentIndex!);
```

Comment:
The `RowViewRef` is actually set in the `DisposeViewRepeaterStrategy` which is a member of the `collection` package so we can't access it, hence the re-get of the `RowViewRef`